lib/matplotlib/figure.py

```diff
@@ -750,19 +755,24 @@ def suptitle(self, t, *, x=.5, y=.98, **kwargs):
             sup.remove()
         else:
             self._suptitle = sup
-        if self._layoutbox is not None:
-            # assign a layout box to the suptitle...
-            figlb = self._layoutbox
-            self._suptitle._layoutbox = layoutbox.LayoutBox(
-                                            parent=figlb,
-                                            name=figlb.name+'.suptitle')
-            for child in figlb.children:
-                if not (child == self._suptitle._layoutbox):
-                    w_pad, h_pad, wspace, hspace =  \
-                            self.get_constrained_layout_pads(
-                                    relative=True)
-                    layoutbox.vstack([self._suptitle._layoutbox, child],
-                                     padding=h_pad*2., strength='required')
+            if self._layoutbox is not None:
+                if not manual_position:
+                    figlb = self._layoutbox
+                    self._suptitle._layoutbox = \
+                            layoutbox.LayoutBox(parent=figlb,
+                                                artist=self._suptitle,
+                                                name=figlb.name+'.suptitle')
+                    for child in figlb.children:
+                        if not (child == self._suptitle._layoutbox):
+                            w_pad, h_pad, wspace, hspace =  \
+                                    self.get_constrained_layout_pads(
+                                            relative=True)
+                            layoutbox.vstack([self._suptitle._layoutbox,
+                                                child],
```

Comment:
slightly weird indentation

Reply:
Oh yeah, I rearranged this a bit so the indentation wasn't as bad.

OTOH, I was purposely stacking children on top of each other for v-stack as a cisual clue they are being vertically stacked.  So I kept that in.